app/controllers/dashboards_controller.rb

```diff
@@ -0,0 +1,5 @@
+class DashboardsController < ApplicationController
```

Comment:
Wondering if this should be a MerchantsController? Or does it have to be this way because of the URLs in the spec?

Reply:
Looks good now after the checkin!